frontend/packages/gitops-plugin/integration-tests/features/gitops/add-manifest-url.feature

```diff
@@ -23,20 +21,34 @@ Feature: Applications managed by GitOps and Create secret
              Then user will see the list GitOps application groupings on the page
 
         @regression @manual
-        Scenario: Application Details page for Applications: GO-01-TC04
+        Scenario: Application Details page for Applications: GO-01-TC03
             Given user is on the Environments page
               And user can see the Applications on the page
              Then user can add new environments using kam cli
 
         @regression @manual
-        Scenario: Application Details page for Applications: GO-01-TC03
+        Scenario: Application Details page for Applications: GO-01-TC04
+            Given user is on the Environments page
+              And user can see the Applications on the page
+             Then user can see various environments for that application
+              And user can see "overview" and "Deployment history" tab for that application
```

Comment:
The Overview and Deployment History tabs are not in this release, we no longer have the Deployment History page going in to this release so the tabs and the link on the card itself to Deployment History are no longer included. This line can be removed

Reply:
Make sense!
Done